modules/ipi-install-preparing-to-deploy-with-virtual-media-on-the-baremetal-network.adoc

```diff
@@ -0,0 +1,119 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+[id="preparing-to-deploy-with-virtual-media-on-the-baremetal-network_{context}"]
+= Preparing to deploy with Virtual Media on the baremetal network
+
+If the `provisioning` network is enabled and you want to expand the cluster using Virtual Media on the `baremetal` network, use the following procedure.
+
```

Comment:
If prerequisites have not already been established, you could add them here. For example, if the user sets up a network named bare metal earlier in the assembly, then it may make sense to keep the backtick styling and a prerequisite to the module to have access to that particular network.

Reply:
Done. Thanks.